src/python/jtmodules/generate_volume_image.py

```diff
@@ -0,0 +1,260 @@
+# Copyright 2017 Scott Berry, University of Zurich
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''Jterator module for extracting a volume image from a 3D stack'''
+import numpy as np
+import collections
```

Comment:
Python's style guide PEP-8 recommends the following order for imports: 1) imports from stdlib, 2) imports from 3rd party libraries, 3) imports from local library.  In this case, it would be: 1) collections + logging, 2) numpy, 3) jtmodules.

Not a big deal, but style checkers (e.g., pylint) and some Python IDEs (e.g. PyCharm) will flag this.